pom.xml

```
@@ -37,6 +37,7 @@
 
     <!-- OptaPlanner version -->
     <version.org.optaplanner>8.6.0-SNAPSHOT</version.org.optaplanner>
+    <version.commons-csv>1.8</version.commons-csv>
```

We usually use a different naming convention
```suggestion
    <version.org.apache.commons.csv>1.8</version.org.apache.commons.csv>
```